### ISSUE TITLE
W3 Validator error

### ISSUE THREAD

**Issue by AuroraStorm-sw** [AuroraStorm-sw/Inspired-Ink#12]
W3 Testing came with 2 errors for 2 pages, 1 for Viewing an Ink and 1 for Editing an Ink.
According to W3, the View Ink site has two stray ending divs, one belonging to the top div class="container-fluid" as seen below;

![InkView](https://res.cloudinary.com/dg4yefryk/image/upload/v1691151150/test-detail_bbnkqd.png)
    ![InkViewCode](https://res.cloudinary.com/dg4yefryk/image/upload/v1691151150/test-detail-code_cevs2o.png)

When scouring my code, I could not find where these errors pointed to, and for the errors to go away, I had to delete open tags that belongs to the top div class="container-fluid", which in turn leaves the div open and so destroys the layout of my comment section.
When validating the same page with a browser addon HTML validator, including W3c Online, these errors were not present:

![W3OnlineResult](https://res.cloudinary.com/dg4yefryk/image/upload/v1691157064/w3online-result_turusx.png)

I have not been able to solve this error, and it does not negatively affect my website in anyway that I can find.

For the Edit Ink page, the W3 error claims that there is an unclosed div in where my background image is wrapped around the block content tags; 

![InkEdit](https://res.cloudinary.com/dg4yefryk/image/upload/v1691151475/test-edit_gkajnl.png)
    ![InkEditCode](https://res.cloudinary.com/dg4yefryk/image/upload/v1691151475/test-edit-code_ioc9tw.png)

The code in question has a closing div;
![bg-img-code](https://res.cloudinary.com/dg4yefryk/image/upload/v1691157301/background-img-code_fahq68.png)

Similarly, when validating the page with the browser extension, there were no errors thrown.
And as with the previous error, I have not been able to solve it, and it does not affect my code in any negative way.

After seeking [assistance on Slack](https://code-institute-room.slack.com/archives/C026PTF46F5/p1691157935163159), I was suggested to install [HTMLHint](https://htmlhint.com/) to scan my HTML files, and the result came out without errors.

![htmlhint](https://res.cloudinary.com/dg4yefryk/image/upload/v1691163151/htmlhint_papoau.png)

To summarize, I have not been able to find a logic reason for W3 Validator to throw these errors, and I will not continue pursuing a solution as my webage works as expected as is, and if I commit to the changes suggested by W3 Validator, my page layout will crash.
